Tests/Functional/Controller/TaskControllerTest.php

```diff
@@ -310,7 +310,8 @@ public function testPut(
             '/api/tasks/' . $postData['id'],
             [
                 'handlerClass' => $handlerClass,
-                'entityId' => $postData['id'],
+                'entityId' => $postData['entityId'],
+                'taskId' => $postData['taskId'],
```

Comment:
@alexander-schranz Can you take a look at these changes?

Reply:
Without those changes, a test fails (See https://github.com/sulu/SuluAutomationBundle/runs/1707833641)

Reply:
That test did also fail on travis (See https://travis-ci.org/github/sulu/SuluAutomationBundle/jobs/730997908)